src/wp-includes/block-template-utils.php

```diff
@@ -532,6 +531,7 @@ function _build_block_template_result_from_file( $template_file, $template_type
 	$template->has_theme_file = true;
 	$template->is_custom      = true;
 	$template->modified       = null;
+	$template->content        = file_get_contents( $template_file['path'] );
```

Comment:
One final suggestion: Let's set `$template->content` right below `$template->theme` (i.e. on R526 if I'm not mistaken).

That's the [position](https://github.com/WordPress/wordpress-develop/pull/5192/files#diff-1d8e05d3ea69557ca8742330e62c1893a72d4ca4ff2ed4ccc5b94bcf5d20fed7L568) where it originally was before we started porting Block Hooks code to Core, and it's also more consistent with `_build_block_template_result_from_post`: https://github.com/WordPress/wordpress-develop/blob/0b8ca16ea3bd9722bd1a38f8ab68901506b1a0e7/src/wp-includes/block-template-utils.php#L760-L762